### PR TITLE
AUSTA-291: Bug in step "Delete Drivers ISO on ESXi Host with Windows Worker"

### LINKED ISSUE #28: Bug in step "Set Boot Order with Windows Worker"
There is a bug in step `Set Boot Order with Windows Worker`.
```
Missing '(' after 'if' in if statement.
```
image

### LINKED ISSUE #30: Bug in step "Delete Drivers ISO on ESXi Host with Windows Worker".
There is a bug in step `Delete Drivers ISO on ESXi Host with Windows Worker`.
```
Set-Location : Cannot find path 'C:\kickstart' because it does not exist.
```
image

### PR DESCRIPTION
**Pull request by h4xhor**
1. Modified the step `Delete Drivers ISO on ESXi Host with Windows Worker` from using the Attune parameter `automationWorkerWindowsBaseDirectory` to using `C:\attuneautomationworker`
2. Modified step `Deploy VMWare Driver on Windows Worker` from using the Attune parameter `automationWorkerWindowsBaseDirectory` to using `/c$/attuneautomationworker`.

closes https://github.com/Attune-Automation/VMWare-ESXi-APIs/issues/30.
closes https://github.com/Attune-Automation/VMWare-ESXi-APIs/issues/28.

### CODE REVIEW

File: steps/deletedriversisoonesxihostwithmacosorlinuxworker/script.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
-cd "{automationWorkerLinuxBaseDirectory}"
+cd "{automationWorkerWindowsBaseDirectory}"
```
Comment (brentonford) [inline steps/deletedriversisoonesxihostwithmacosorlinuxworker/script.txt]: Replace the `{automationWorkerWindowsBaseDirectory}` with `C:\attuneautomationworker`

File: steps/deletedriversisoonesxihostwithmacosorlinuxworker/script.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
-cd "{automationWorkerLinuxBaseDirectory}"
+cd "{automationWorkerWindowsBaseDirectory}"
 if [ ! -f drivers_{newVmNode.fqn}.iso ]
 then
-    echo "This build doesn't have a {automationWorkerLinuxBaseDirectory}/drivers_{newVmNode.fqn}.iso."
+    echo "This build doesn't have a {automationWorkerWindowsBaseDirectory}/drivers_{newVmNode.fqn}.iso."
```
Comment (brentonford) [inline steps/deletedriversisoonesxihostwithmacosorlinuxworker/script.txt]: Replace the `{automationWorkerWindowsBaseDirectory}` with `C:\attuneautomationworker`

**Review by brentonford — CHANGES_REQUESTED** [review]
@h4xhor you missed a `{automationWorkerWindowsBaseDirectory}` Parameter to be replaced with `C:\attuneautomationworker`. Please remove the `{automationWorkerWindowsBaseDirectory}` Parameter.

File: steps/deletedriversisoonesxihostwithmacosorlinuxworker/script.txt (hunk 2 above)
Comment (jarrodchesney) [reply steps/deletedriversisoonesxihostwithmacosorlinuxworker/script.txt]: Oh no. Is that the directory name?

Why wasn't it make more readable like C:\attune_automation_worker ?

Comment (h4xhor) [reply steps/deletedriversisoonesxihostwithmacosorlinuxworker/script.txt]: @brentonford and @jarrodchesney I just realized it's my error. The step is for macOS or Linux Worker so it should be using `{automationWorkerLinuxBaseDirectory}`.

Comment (h4xhor) [reply steps/deletedriversisoonesxihostwithmacosorlinuxworker/script.txt]: Hi @brentonford here is the commit `36072b1c0fd99d7d231adaebfc627bfa08ec0ad9` with the correction to my errors:
image

**Pull request merged**